scenarios/crowdsecurity/http-bf-wordpress_bf.yaml

```diff
@@ -6,9 +6,9 @@ debug: false
 filter: "evt.Meta.log_type == 'http_access-log' && evt.Parsed.file_name == 'wp-login.php' && evt.Parsed.verb == 'POST'"
 groupby: evt.Meta.source_ip
 capacity: 5
-leakspeed: "10s"
```

Comment:
Same here (no change needed)

Reply:
reverted in d42f77ca6e0ae9d4d537818eb50d60f72f17c83a